cmd/ike/cmd/develop.go

```diff
@@ -107,6 +109,7 @@ func NewDevelopCmd() *cobra.Command {
 	developCmd.Flags().Bool("no-build", false, "always skips build")
 	developCmd.Flags().StringSliceP("watch", "w", []string{currentDir()}, "list of directories to watch")
 	developCmd.Flags().StringSlice("watch-exclude", excludeTpLog, "list of patterns to exclude (defaults to telepresence.log which is always excluded)")
```

Comment:
how about *.log? But I guess default should be .gitignore?

Reply:
Yeah, I was more thinking about handling `.gitignore` and assume it's there. But we can always have set of defaults regardless. That makes sense.